packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html

```diff
@@ -26,289 +26,457 @@
+                                        storageMaxSize: ($ctrl.model.flavor.maxDiskSize | bytes:2:false:'GB')
+                                    }"
+                                ></span>
+                            </div>
+                        </div>
+                        <div class="mt-2">
+                            <a
+                                href=""
+                                data-ng-click="$ctrl.scrollTo('network')"
+                                data-translate="pci_database_add_spec_network"
+                            ></a>
+                            <span
+                                data-ng-if="$ctrl.model.usePrivateNetwork"
+                                data-translate="pci_database_add_network_mode_private"
+                            ></span>
+                            <span
+                                data-ng-if="!$ctrl.model.usePrivateNetwork"
+                                data-translate="pci_database_add_network_mode_public"
+                            ></span>
+                        </div>
+                        <div
+                            class="d-flex flex-column mb-2 pl-2"
+                            data-ng-if="$ctrl.model.usePrivateNetwork "
+                        >
+                            <div class="d-flex">
+                                <span
+                                    class="oui-icon oui-icon-cloud_concept mr-2"
+                                ></span>
+                                <span
+                                    data-translate="pci_database_add_spec_network_name"
+                                    class="mr-2"
+                                ></span>
+                                <span
+                                    data-ng-if="$ctrl.model.privateNetwork.id.length !== 0"
+                                    data-ng-bind="$ctrl.model.privateNetwork.name"
+                                ></span>
+                                <span
+                                    class="oui-color-ae-500"
+                                    data-ng-if="$ctrl.model.privateNetwork.id.length === 0"
+                                    data-translate="pci_database_add_spec_network_none"
+                                ></span>
+                            </div>
+                            <div class="d-flex">
+                                <span
+                                    class="oui-icon oui-icon-cloud-edge-computing_concept mr-2"
+                                ></span>
+                                <span
+                                    data-translate="pci_database_add_spec_network_subnetwork"
+                                    class="mr-2"
+                                ></span>
+                                <span
+                                    data-ng-if="$ctrl.model.subnet.id.length !== 0"
+                                    ng_bind="$ctrl.model.subnet.name"
```

Comment:
Big mistake here, `ng_bind` don't existe :)
```suggestion
                                    data-ng-bind=":: $ctrl.model.subnet.name"
```